forge-gui/res/cardsfolder/upcoming/hew_the_entwood.txt

```diff
@@ -0,0 +1,16 @@
+Name:Hew the Entwood
+ManaCost:3 R R
+Types:Sorcery
+A:SP$ Sacrifice | Defined$ You | Amount$ SacX | SacValid$ Land | RememberSacrificed$ True | SubAbility$ DBReveal | SpellDescription$Sacrifice any number of lands. Reveal the top X cards of your library, where X is the number of lands sacrificed this way. Choose any number of artifact and/or land cards revealed this way. Put all nonland cards chosen this way onto the battlefield, then put all land cards chosen this way onto the battlefield tapped, then put the rest on the bottom of your library in a random order.
```

Comment:
how you handle "any number"?

Reply:
I just added optional$ True which is the param that you find in similar cards (like shapeshift)